core/src/main/java/org/pentaho/agilebi/modeler/nodes/AbstractMetaDataModelNode.java

```diff
@@ -177,6 +186,49 @@ public void setClassname( String classname ) {
     }
   }
 
+  @Bindable
+  public String getAltText() {
+    return altText;
+  }
+
+  @Bindable
+  public void setAltText( String altText ) {
+    if ( this.altText == null || !this.altText.equals( altText ) ) {
+      String oldAltText = this.altText;
+      this.altText = altText;
+      if ( !suppressEvents ) {
+        this.firePropertyChange( "altText", oldAltText, altText ); //$NON-NLS-1$
+      }
+    }
+  }
+
+  /**
+   * Retrieve <code>propertyName</code> from modeler message property and then set alternative text.
+   * @param propertyName
+   */
+  protected void getMessageStringAndSetAltText( String propertyName ) {
```

Comment:
helper function to avoid DRY for grabbing value from property file and calling #setAltText()